presto-main/src/main/java/com/facebook/presto/util/Failures.java

```diff
@@ -55,21 +57,26 @@ public static ExecutionFailureInfo toFailure(Throwable failure)
         }
         // todo prevent looping with suppressed cause loops and such
         String type;
+        HostAddress destinationHost = null;
```

Comment:
@raghavsethi : For class field, yes. For local variable, it doesn't seem so. (Compiler will complain "Variable xxx might not have been initialized" :) )